augly/image/transforms.py

```diff
@@ -2386,6 +2397,7 @@ def apply_transform(
             emoji_size=self.emoji_size,
             x_pos=self.x_pos,
             y_pos=self.y_pos,
+            seed=self.seed,
```

Comment:
shouldn't you set the random seed on line 2392, i.e. where we randomly choose which emoji to use?

Reply:
Agreed, @lyakaap when you move all the param setting to line 2392, please make sure to set the seed before line 2392 so that *all* the param sampling is reproducible using the seed.